code/modules/clothing/head/armor.dm

```diff
@@ -581,6 +581,31 @@
 	name = "gray tanker helmet"
 	icon_state = "tanker_helmet_gray"
 
+/obj/item/clothing/head/armor/excel_shock
+	name = "Excelsior Shock Trooper Helmet"
+	desc = "Reminisciant of earlier ablative designs, this shock helmet is at the forefront of haven's battles."
+	icon_state = "shock_helm"
+	item_state = "shock_helm"
+	armor = list(
+		melee = 8,
+		bullet = 13,
+		energy = 10,
+		bomb = 25,
+		bio = 0,
+		rad = 0
+	)
+	matter = list(
+	MATERIAL_PLASTIC = 2,
+	MATERIAL_GLASS = 2,
+	MATERIAL_STEEL =3,
+	MATERIAL_PLASTEEL
```

Comment:
MATERIAL_PLASTEEL? You forgot to include the amount of it. If it's '1', you should still specify it.